src/v/redpanda/application.cc

```diff
@@ -428,9 +482,17 @@ void application::wire_up_services() {
                            .get_credentials_builder()
                            .get0();
     kafka_cfg.credentials
-      = kafka_builder
-          ? kafka_builder->build_reloadable_server_credentials().get0()
-          : nullptr;
+      = kafka_builder ? kafka_builder
+                          ->build_reloadable_server_credentials(
+                            [](
+                              const std::unordered_set<ss::sstring>& updated,
+                              const std::exception_ptr& eptr) {
+                                static ss::logger certlog("krpc-tls");
```

Comment:
can we use existing loggers here instead of building complete new loggers only for tls events? i think using the existing rpc logger or kafka logger would make sense. same for the other cases.

Reply:
done